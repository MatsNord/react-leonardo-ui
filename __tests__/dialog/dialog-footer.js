import React from 'react';
import renderer from 'react-test-renderer';

import Dialog from '../../src/dialog';

test('Dialog footer renders correctly', () => {
  const tree = renderer.create(
    <Dialog.Footer>Footer content</Dialog.Footer>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Dialog footer passes props correctly', () => {
  const tree = renderer.create(
    <Dialog.Footer className="my-class" data-x="x" title="Footer title">
      Footer title
    </Dialog.Footer>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
