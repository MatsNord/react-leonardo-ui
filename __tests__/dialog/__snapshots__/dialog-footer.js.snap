// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog footer passes props correctly 1`] = `
<div
  className="lui-dialog__footer my-class"
  data-x="x"
  title="Footer title"
>
  Footer title
</div>
`;

exports[`Dialog footer renders correctly 1`] = `
<div
  className="lui-dialog__footer"
>
  Footer content
</div>
`;
